frontend/src/components/ui-kit/components/Checkbox/Checkbox.module.scss

```diff
@@ -1,28 +1,16 @@
 @import '../../../../variables';
+
 .checkbox {
-  max-width: 280px;
-  margin: 50px;
-}
-label {
   display: flex;
   align-items: center;
-  gap: 10px;
-  margin-left: 30px;
-}
-hr {
-  background-color: $gray-light-color;
-  margin-top: 16px;
-}
-.selectedOptions {
-  position: absolute;
-  left: 30%;
-  top: 25%;
-}
-.selectedItem {
-  display: inline-flex;
-  align-items: center;
-  background-color: $page-background-color;
-  margin: 12px;
-  padding: 6px 12px 6px 8px;
-  border-radius: 8px;
+  gap: 8px;
+
+  input {
+    display: none;
+  }
+
+  .label {
+    color: $text-color;
+    margin-bottom: 5px;
```

Comment:
Please remove this and center it, now everything is pushed to the top, not in the center